source/dlangbot/github.d

```diff
@@ -217,13 +217,17 @@ Json[] tryMerge(in ref PullRequest pr, GHMerge.MergeMethod method)
         return commits;
     }
 
-    auto labelName = method.labelName;
+    const labelName = method.labelName;
+    const mergeLabelName = "Merge:" ~ labelName;
     if (commits.length == 1)
         method = GHMerge.MergeMethod.rebase;
 
     auto events = ghGetRequest(pr.eventsURL).body[]
         .retro
-        .filter!(e => e["event"] == "labeled" && e["label"]["name"] == labelName);
+        .filter!(e => e["event"] == "labeled" && (
+                 e["label"]["name"] == labelName)
```

Comment:
I think this `)` should be a few lines lower.

Reply:
I suggest using `std.algorithm.among`, e.g.: `e["label"]["name"].among(labelName, mergeLabelName)`

Reply:
Done